### ISSUE TITLE
Python-path of Docker installation iof DOLFINx hardcoded

### ISSUE THREAD

**Issue by jorgensd** [FEniCS/dolfinx#1847]
The path is hardcoded to `python3.8` which does not make sense as we are using `python3.9`
https://github.com/FEniCS/dolfinx/blob/810baf3861366cda3d6d31a3cff33e6518aec6be/docker/Dockerfile#L420-L428

**Comment by jhale** [comment]
Good catch, a bit ugly although the naming is 'arbitrary' in this context. Rather than overcomplicate things by calling the Python intepreter to get our the major minor version, we can simply switch to using lib/python/dist-packages.
